src/shogun/lib/any.cpp

```diff
@@ -0,0 +1,28 @@
+#include <shogun/lib/any.h>
+
+template<>
+Any::EnumType Any::type2enum<int>()
```

Comment:
oh i **was** trying to use `policy.type()` which only gives only int or double. I can be more specific if this strategy is okay